src/resources/js/v2/tickets-commerce.js

```diff
@@ -180,6 +183,28 @@ tribe.tickets.commerce = {};
 		$document.trigger( 'afterSetup.tecTicketsCommerce', [ $container ] );
 	};
 
+	/**
+	 * Get purchaser data if available.
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container Container for the purchaser info fields.
+	 *
+	 * @return {Object}
+	 */
+	obj.getPurchaserData = function ( $container ) {
+		let purchaser = {};
```

Comment:
Why are we using `let` here?

Reply:
No specific reason should be `const` now.